specification/components/schemas/extensions/ServiceSummaryViewExtension.yaml

```diff
@@ -0,0 +1,473 @@
+required:
+  - url
+  - extension
+properties:
+  url:
+    type: string
+    enum:
+      - 'https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ServiceSummaryView-1'
+    example: 'https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ServiceSummaryView-1'
+  extension:
+    type: array
+    items:
+      anyOf:
+        - title: Supported appointment type
+          type: object
+          required:
+            - url
+            - valueCodeableConcept
+          properties:
+            url:
+              type: string
+              enum:
+                - supportedAppointmentType
+              example: 'supportedAppointmentType'
+            valueCodeableConcept:
+              type: object
+              required:
+                - coding
+              properties:
+                coding:
+                  type: array
+                  minItems: 1
+                  maxItems: 1
+                  items:
+                    type: object
+                    required:
+                     - system
+                     - code 
+                    properties:
+                      system:
+                        type: string
+                        enum:
+                          - https://fhir.nhs.uk/STU3/CodeSystem/eRS-AppointmentType-1
+                        example: 'https://fhir.nhs.uk/STU3/CodeSystem/eRS-AppointmentType-1'
+                      code:
+                        type: string
+                        enum: 
+                          - DAY_CASE
+                          - FIRST_OUTPATIENT
+                          - ASSESSMENT_SERVICE
+                          - GPSI
+                          - AHP_CLINIC
+                          - DIAGNOSTIC
+                          - NURSE_CLINIC
+                          - OPHTHALMOLOGIST
+                          - TELEPHONE_ASSESSMENT
+                        example: 'DIAGNOSTIC'
+        - title: Name of the service.
+          type: object
+          required:
+            - url
+            - valueString
+          properties:
+            url:
+              type: string
+              enum:
+                - serviceName
+              example: 'serviceName'
+            valueString:
+              type: string
+              example: 'Good Cardiology Business Service'
+        - title: Services can support one or two Request Flow Types, but not Triage Request and Appointment Request simultaneously.
+          type: object
+          required:
+            - url
+            - valueCodeableConcept
+          properties:
+            url:
+              type: string
+              enum:
+                - supportedRequestFlowType
+              example: 'supportedRequestFlowType'
+            valueCodeableConcept:
+              type: object
+              required:
+                - coding
+              properties:
+                coding:
+                  type: array
+                  minItems: 1
+                  maxItems: 1
```

Comment:
Shouldn't this maxItems be 2?

Reply:
Confirmed, 1 is the right value in this instance